### PR TITLE
Disable warnings for C++ compilation of messages.proto

### PR DESCRIPTION
**Pull request by JeanPaulShapo**

### CODE REVIEW

**Comment by JeanPaulShapo** [comment]
Fixes #355 
@sashafrey I ask you to check these changes using Visual Studio.

File: src/artm/CMakeLists.txt [hunk 1]
```diff
@@ -2,29 +2,38 @@ include_directories(${CMAKE_CURRENT_LIST_DIR}/..)
...
+set(GENERATED_PROTO_FILES ${GENERATED_PROTO_FILES}
+  ${CMAKE_CURRENT_LIST_DIR}/messages.pb.cc
+  ${CMAKE_CURRENT_LIST_DIR}/messages.pb.h
+  ${CMAKE_CURRENT_LIST_DIR}/core/internals.pb.cc
+  ${CMAKE_CURRENT_LIST_DIR}/core/internals.pb.h)
+    
+set_source_files_properties(${GENERATED_PROTO_FILES} PROPERTIES GENERATED TRUE)
+
+if (MSVC)
+  set_source_files_properties(${GENERATED_PROTO_FILES} PROPERTIES COMPILE_FLAGS "/wd4244" "/wd4267")
```
Comment (sashafrey) [inline src/artm/CMakeLists.txt]: Remove ``"`` between two ``/wd`` switches. Correct line is
```
set_source_files_properties(${GENERATED_PROTO_FILES} PROPERTIES COMPILE_FLAGS "/wd4244 /wd4267")
```

**Comment by sashafrey** [comment]
@JeanPaulShapo I've added one comment, which fixed the build on my machine. But I had impression that it (maybe?) increased the number of warnings during Windows build, and I couldn't find time to look into this closer. Is it OK if this PR stays open for a while? I'd like to test it a little bit more on Windows.

**Comment by JeanPaulShapo** [comment]
@sashafrey Yes, it's OK. You can test it as long as you need.

**Pull request closed without merging**